github_issue_summarization/02_distributed_training.md

```diff
@@ -1,5 +1,10 @@
 # Distributed training using Estimator
 
+Distributed training with keras currently doesn't work; see
+
+* kubeflow/examples#280
```

Comment:
You're training with Keras + tf-estimator, just not tf.keras. Is this still accurate?

Reply:
I don't know how to run distributed training without TF.Estimator. 

The code using TF.Estimator doesn't work. So right now we don't have a way to run distributed training.